crates/languages/src/dart/folds.scm

```diff
@@ -0,0 +1,12 @@
+[
+  (class_definition)
+  (enum_declaration)
+  (extension_declaration)
+  (arguments)
+  (function_body)
+  (block)
+  (switch_block)
+  (list_literal)
+  (set_or_map_literal)
+  (string_literal)
+] @fold
```

Comment:
Zed currently doesn't use a query called `folds.scm`. Can you remove this?

Reply:
Oh. I saw folds schema added to languages like Zig, Erlang and Svelte.


<img width="554" alt="image" src="https://github.com/zed-industries/zed/assets/25414681/2a0355ef-798f-4d37-8627-91e183a412e2">


I've removed it as you suggested, but let me know, if we can have it again.